packages/hint/docs/contributor-guide/getting-started/pull-requests.md

```diff
@@ -75,25 +75,21 @@ git commit
 Commit messages should have the following structure:
 
 ```text
-<Tag>: Capitalized, summary (less then 50 chars)
-<blank line>
-If necessary, more detailed explanatory text, wrapped to about 72
-characters or so.
+<Tag>: Capitalized, summary (50 chars recommended)
 
-The commit message should be in the imperative: "Fix bug" and not
-"Fixed bug" or "Fixes bug."  This convention matches up with commit
-messages generated by commands like `git merge` and `git revert`.
+If necessary, more detailed explanatory text and/or what this commit
+fixes, wrapped to about 72 characters or so. The commit message should
+be in the imperative: "Fix bug" and not "Fixed bug" or "Fixes bug."
 
-If applicable, at the end, write where the users can find more
-information (URL(s), or other related issues), and/or what this
-commit fixes.
+If applicable, to refer to an issue, comment or URL not being resolved
```

Comment:
Ah, I think I wasn't clear. I was thinking of putting the info about `Ref` in the text outside of the example after
> **All other lines**, where possible, should be wrapped to about 72 characters.

That way we keep the example in the documentation matching the commit message template.

Reply:
Do you mean that we include the info about `Ref` in the same para, maybe after
> where possible, should be wrapped to about 72 characters.

And have the examples

> For example:
`Ref #1911`
`Ref https://github.com/webhintio/hint/pull/1911#issuecomment-463715363`
`Ref https://webhint.io/docs/contributor-guide/`

at the end of this para?

> That way we keep the example in the documentation matching the commit message template.

My understanding is we are updating both the commit message template at [`prepare-commit-message.js`](https://github.com/webhintio/hint/blob/master/scripts/prepare-commit-message.js#L11) and the example at [`pull-request.md`](https://github.com/webhintio/hint/blob/master/packages/hint/docs/contributor-guide/getting-started/pull-requests.md#step-2-make-your-changes), to match the new simpler template we decided: 
```
const NEW_GIT_COMMIT_MESSAGE = `
### Please follow commit style. Before committing see:
### https://webhint.io/docs/contributor-guide/getting-started/pull-requests/#commit-messages
###
# <Tag>: Capitalized, summary (50 chars recommended)
#
# If necessary, more detailed explanatory text and/or what this commit
# fixes, wrapped to about 72 characters or so. The commit message should
# be in the imperative: "Fix bug" and not "Fixed bug" or "Fixes bug."
#
# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
#
# Fix #<issue>
`;
```
Let me know where I miss your point.

Reply:
> Do you mean that we include the info about `Ref` in the same para, maybe after
> > where possible, should be wrapped to about 72 characters.

You got it.

> My understanding is we are updating both the commit message template at [`prepare-commit-message.js`](https://github.com/webhintio/hint/blob/master/scripts/prepare-commit-message.js#L11) and the example at [`pull-request.md`](https://github.com/webhintio/hint/blob/master/packages/hint/docs/contributor-guide/getting-started/pull-requests.md#step-2-make-your-changes), to match the new simpler template we decided:

Yes, exactly.

Reply:
Okay, got it. You mean adding info about `Ref` in the paragraphs after the example in pull-requests.md. Specifically after 
> **All other lines**, where possible, should be wrapped to about 72 characters.
And not including it in the template, as I have done. Sorry, my bad. Will remove it from both the example and the `prepare-commit-message.js` template, and add it in the documentation below the example.

Reply:
Sorry, deleted the earlier comment. It felt too long, and confusing. This one captures it. Sorry quite rusty at working on Github.